kairos-test-utils/Cargo.toml

```diff
@@ -16,4 +16,5 @@ nom = "7"
 tokio = { version = "1", features = [ "full", "tracing", "macros" ] }
 tempfile = "3"
 tracing = "0.1"
+reqwest = { version = "*", features = ["json"] }
```

Comment:
What do we gain from version="*" here? I don't think it matters but I would like to understand :).

Reply:
There should not be bare wildcard here.

Reply:
I allow cargo to figure out the right version. The build/test determines whether my code continues to be correct. I don't have to pin semantic versions. If for some reason I run `cargo update` and a newer reqwest version is selected I fix the errors.

Reply:
Pinning a specific version just causes multiple package versions being locked by cargo instead of allowing cargo to figure out a common base.

Reply:
I don’t mind loose version bounds in an app or in this case a test lib, but my preference would be for most significant non zero SemVer component.

So in this case I'd prefer “0.12”.

Reply:
I require `0.12` now [c585c51](https://github.com/cspr-rad/kairos/pull/49/commits/c585c512f7db2e34d0c987ddbaccaa3350c825cf)